wire-library/wire-tests-swift/src/main/swift/Form.swift

```diff
@@ -207,6 +207,16 @@ extension Form.Decision : Hashable {
 }
 #endif
 
+#if !WIRE_REMOVE_REDACTABLE
```

Comment:
?

Reply:
That's generated automatically by `generateTests`, it's already in a few places.